### ISSUE TITLE
Gimbled weaponry doesn't apply to logic driven weapons

### ISSUE THREAD

**Issue by Xenoprophet-Nail** [tsunamayo/Starship-EVO#2800]
### Short Description of the issue:
I use a series of delays to fire 8 small lasers with heat barrels, after giving the experimental branch a try I noticed my new ship doesn't aim at targets with them, I wonder if this is intentional as a few people do use that method for rapid fire weaponry. 
I hope to hear an answer soon, I'm loving the medbay by the way it's real cool
...

### Steps to reproduce the issue:
connect a yoke to a weapon hotkey, the weapon hotkey to the first laser and to a delay of .1 that delay is connected to another laser (for the sake of the test I think three will do) the delay goes to another delay and to a laser
...

### Environment
- Starship EVO version:  Early-Access 20w35a
- Graphics Card: Nvidia GTX 1060 6gb

### Additional information:
[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/5116821/output_log.txt)
![20200824050126_1](https://user-images.githubusercontent.com/51257864/91025193-08604180-e5c7-11ea-8738-65ce263a2278.jpg)
![20200824050114_1](https://user-images.githubusercontent.com/51257864/91025197-0a2a0500-e5c7-11ea-935b-4c8d9486f893.jpg)
yeah my logic is crammed into something the size of a dalek lol the orange is the setup for the lasers, my two beam weapons (those work) and an or gate I painted orange accidentally which is for my reactor's animation.

_(Screenshots, Videos, Error logs, Related issues)_

Please also include the `output_log.txt` file. You can find your log here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

**Comment by tsunamayo** [comment]
Yes I saw that on a ship too. I am not sure I will be able to support that...

**Comment by Xenoprophet-Nail** [comment]
shame, well I won't press the situation further. Thank you for the reply

**Comment by tsunamayo** [comment]
yeah I will give it a shot, this is a bit tricky (and messy ;))

**Comment by Xenoprophet-Nail** [comment]
Okay, well if it doesn't work I'll figure something out. You can take your time with it, if you're pressed for other things.  I can be patient. though sometimes I'm not lol I made a walking machine in sevo that walks but is slow, I have a gif but it collides with me while I'm filming it and doesn't turn right. I managed to overcome my enemy hills.. >:3
https://discordapp.com/channels/@me/747691692345393153/747793310122246215

**Comment by tsunamayo** [comment]
@Xenoprophet-Nail damn amazing send me the bp!

**Comment by Xenoprophet-Nail** [comment]
No problem, best to hit F5 before playing with it as it may collide with you, don't want it to not turn a direction like it did in that gif.., I have noticed it's started sliding along the ground as of the experimental updates.. lol Build a walking thing without friction can have those issues though. Someone suggested to me to bring up some form of magnetism for walker feet so that you don't have to go crazy with a friction script. If you intend to add tank treads and wheels some time (in the far future) you'd probably still need friction lol.. eh just sharing someone's idea.. 

Anyhoo I ramble too much lol here ya go:
![Walker Skritter IV_preview](https://user-images.githubusercontent.com/51257864/91631289-96ad3c80-e9a6-11ea-9a1f-5d3647e57ed7.png)
[Walker Skritter IV.zip](https://github.com/tsunamayo/Starship-EVO/files/5144879/Walker.Skritter.IV.zip)
